tests/validation/cattlevalidationtest/core/test_ldap.py

```diff
@@ -0,0 +1,181 @@
+from common_fixtures import *  # NOQA
+from requests.auth import AuthBase
+
+
+if_ldap = pytest.mark.skipif(not os.environ.get('API_AUTH_LDAP_SERVER'),
+                             reason='API_AUTH_LDAP_SERVER is not set')
+
+
+BASE_URL = cattle_url()
```

Comment:
Ok, should we abstract the whole base_url logic out into a common function then? Appears this one is more robust than githubs.